style.css

```
@@ -0,0 +1,34 @@
+@charset "UTF-8";
+
+/*--------- 共通部分 ---------*/
+html{
+    font-size: 0.625rem;
+}
+
+body{
+    color: #3c3c3c;
+    background-color: rgb(251, 244, 235);
+    line-height: 1.7;
+    font-size: 1.4rem;
+    letter-spacing: 0.1em;
+    font-family: 'Raleway', sans-serif;
```

```suggestion
    font-family: "Raleway","Hiragino Kaku Gothic ProN","Hiragino Sans","Helvetica Neue",Arial,Meiryo,sans-serif;
```
日本語サイトの場合はfont-familyを日本語ベースでセットしないとダメです。